sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -516,6 +516,25 @@ public void testExecuteBatchWithOneFailingAndNothingExecuted() throws Exception
         }
     }
 
+    @Test
+    public void test_execute_bulk_insert_from_subquery_fails_with_analyzer__types_incompatibility_exception() throws Exception {
```

Comment:
Would it be possible to turn this into an analyzer test? (Cheaper to run)

Reply:
i deliberately wanted it to keep as an integration test, but probably it should be also fine to have it as the . analyzer test.  👍 will update